opal-rspec.gemspec

```diff
@@ -15,9 +15,7 @@ Gem::Specification.new do |s|
 
   s.require_paths  = ['lib']
 
-  s.add_dependency 'opal', ['>= 0.8.0.rc1', '< 0.9']
-  # Opal now does not include minitest but we need it for Sprockets/test execution
-  s.add_dependency 'opal-minitest', '~> 0.0'
+  s.add_dependency 'opal', ['>= 0.8.0']
```

Comment:
This would be better of with `['>= 0.8', '< 0.10']`, as we can easily change it with a point release if any next version of opal (eg. `0.10` or `1.0`) is compatible.